src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -574,12 +568,11 @@ function updateConstSumQnInstructions(qnNum) {
 
     instruction += `${infoIcon} Total points distributed should add up to ${points}.<br>`;
 
-    // Check if allocating different points to just one recipient/option will be enough
     if (forceSomeUnevenDistribution) {
         instruction += `${infoIcon} At least one ${distributeToRecipients ? 'recipient' : 'option'} `
                 + 'should be allocated different number of points.<br>';
     } else if (forceUnevenDistribution || forceAllUnevenDistribution) {
-        // Each recipient/option must be allocated different points otherwise
+    // All points need to be different. Use forceUnevenDistribution for older versions to check the same
```

Comment:
umm so this is how i have explained it atm. Let me know if something more needs to go in here.